collections/luci-light/Makefile

```diff
@@ -9,15 +9,10 @@ include $(TOPDIR)/rules.mk
 LUCI_TYPE:=col
 LUCI_BASENAME:=light
 
-LUCI_TITLE:=LuCI interface with Uhttpd as Webserver (light)
-LUCI_DESCRIPTION:=Light OpenWrt set including admin support and the default Bootstrap theme
+LUCI_TITLE:=LuCI Configuration Interface with light Uhttpd as webserver
```

Comment:
_Uhttpd_, _uhttpd_ or _uHTTPd_?
It seems [official docs](https://openwrt.org/docs/guide-user/services/webserver/uhttpd) favour the latter.

Reply:
it was already written here as Uhttpd so I kept it as is. It's a good idea to change to be the same as in docs